src/extension.ts

```diff
@@ -71,6 +71,7 @@ export function activate(context: vscode.ExtensionContext): void {
 
     const actionHandler: AzureActionHandler = new AzureActionHandler(context, outputChannel, reporter);
     actionHandler.registerCommand('appService.Refresh', async (node?: IAzureNode) => await tree.refresh(node));
+    vscode.commands.registerCommand('appService.selectSubscriptions', () => vscode.commands.executeCommand("azure-account.selectSubscriptions"));
```

Comment:
Are you intentionally avoiding the actionHandler because you don't want telemetry on this?

Reply:
No, hadn't intended that. Good point, I'll change.